framework/wazuh/rbac/orm.py

```diff
@@ -2662,59 +2663,29 @@ def insert_data_from_yaml(self, database: str, resource_type: ResourceType = Res
                 for d_rule_name, payload in default_rules[next(iter(default_rules))].items():
                     rum.add_rule(name=d_rule_name, rule=payload['rule'], resource_type=resource_type,
                                  check_default=False)
 
-        # Create default policies if they don't exist yet
+        # Create default policies
         with open(os.path.join(default_path, "policies.yaml"), 'r') as stream:
             default_policies = yaml.safe_load(stream)
-
             with PoliciesManager(self.sessions[database]) as pm:
                 for d_policy_name, payload in default_policies[next(iter(default_policies))].items():
                     for name, policy in payload['policies'].items():
-                        policy_name = f'{d_policy_name}_{name}'
-                        policy_result = pm.add_policy(name=policy_name, policy=policy, resource_type=resource_type,
-                                                      check_default=False)
-                        # Update policy if it exists
-                        if policy_result == SecurityError.ALREADY_EXIST:
-                            try:
-                                policy_id = pm.get_policy(policy_name)['id']
-                                if policy_id < max_id_reserved:
-                                    pm.update_policy(policy_id=policy_id, name=policy_name, policy=policy,
-                                                     resource_type=resource_type, check_default=False)
-                                else:
-                                    with RolesPoliciesManager() as rpm:
-                                        linked_roles = [role.id for role in
-                                                        rpm.get_all_roles_from_policy(policy_id=policy_id)]
-                                        new_positions = dict()
-                                        for role in linked_roles:
-                                            new_positions[role] = [p.id for p in
-                                                                   rpm.get_all_policies_from_role(role_id=role)] \
-                                                .index(policy_id)
-
-                                        pm.delete_policy(policy_id=policy_id)
-                                        pm.add_policy(name=policy_name, policy=policy, resource_type=resource_type,
-                                                      check_default=False)
-                                        policy_id = pm.get_policy(policy_name)['id']
-                                        for role, position in new_positions.items():
-                                            rpm.add_role_to_policy(policy_id=policy_id, role_id=role, position=position,
-                                                                   force_admin=True)
+                        pm.add_policy(name=f'{d_policy_name}_{name}', policy=policy, check_default=False)
```

Comment:
```suggestion
                        pm.add_policy(name=f'{d_policy_name}_{name}', policy=policy, resource_type=resource_type, check_default=False)
```

This was adding default policies as `user`.

Reply:
Fixed